specification/machinelearning/resource-manager/Microsoft.MachineLearning/preview/2016-05-01-preview/commitmentPlans.json

```diff
@@ -16,6 +16,31 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearning/skus": {
+      "get": {
+        "tags": [
+          "Skus"
+        ],
+        "operationId": "Skus_List",
+        "description": "Lists the available commitment plan SKUs.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
please mark this operations as
``` JSON
        "x-ms-pageable": {
          "nextLinkName": null
        },
```
In generated SDKs, this will hide the fact that what you return is an object containing an array - all users see will be the array.

Reply:
Done